CHANGES.md

```diff
@@ -24,6 +24,12 @@ OpenSSL 3.1
 
 ### Changes between 3.0 and 3.1 [xx XXX xxxx]
 
+ * Add ciphersuites based on DHE_PSK (RFC 4279) and ECDHE_PSK (RFC 5489)
+   to the list of ciphersuites providing Perfect Forward Secrecy as
+   required by SECLEVEL >= 3.
+
+   *Dmitry Belyavskiy, Nicola Tuveri*
```

Comment:
Not sure about the order because you did almost all work.

Reply:
I stand by it: you reported the issue, proposed its resolution to the OTC, and the work we did independently to implement the actual change is virtually identical.